README.md

```diff
@@ -174,6 +174,7 @@ For more information on the structure of the YAML configuration file, please vis
 | `targetPort`              | No       | The designated port for the application to run on. If no value is provided and the builder is used to build the runnable application image, the target port will be set to 80 for Python applications and 8080 for all other platform applications. If no value is provided when creating a Container App, the target port will default to 80. Note: when using this action to update a Container App, the target port may be updated if not provided based on changes to the ingress property. |
 | `location`                | No       | The location that the Container App (and other created resources) will be deployed to. To view locations suitable for creating the Container App in, please run the following: `az provider show -n Microsoft.App --query "resourceTypes[?resourceType=='containerApps'].locations"` |
 | `environmentVariables`    | No       | A list of environment variable(s) for the container. Space-separated values in 'key=value' format. Empty string to clear existing values. Prefix value with 'secretref:' to reference a secret. |
+| `buildArguments`          | No       | A list of build argument(s) for the container. Space-separated values in 'key=value'.|

```

Comment:
How do you feel about renaming this input to `dockerfileBuildArguments`? I just want to make sure it's explicit that the arguments that we're providing to this input are going to _only_ be used when we call `docker build` when the user provides a `Dockerfile` in their source application. We just need a name that differentiates it from the "builder" flow that will use our buildpacks to build the provided source application.

Reply:
I think at a minimum, even if we don't rename the input, we should update the description to call out that it's for `--build-args` arguments in the `docker build` command that's executed when a `Dockerfile` is provided or found in the user's provided source application.